database/sql/src/main/scala/cromwell/database/slick/CallCachingSlickDatabase.scala

```diff
@@ -19,8 +19,8 @@ trait CallCachingSlickDatabase extends CallCachingSqlDatabase {
                              (implicit ec: ExecutionContext): Future[Unit] = {
 
     // Construct parallel lists of parent entries, hashes, simpletons, and detritus from `CallCachingJoin`s.
-    val (entries, hashes, simpletons, detritus) = joins.toList.foldMap { j =>
-      (List(j.callCachingEntry), List(j.callCachingHashEntries), List(j.callCachingSimpletonEntries), List(j.callCachingDetritusEntries)) }
+    val (entries, hashes, simpletons, detritus, aggregations) = joins.toList.foldMap { j =>
+      (List(j.callCachingEntry), List(j.callCachingHashEntries), List(j.callCachingSimpletonEntries), List(j.callCachingDetritusEntries), List(j.callCachingAggregationEntry.toList)) }
```

Comment:
(This change was part of my rebase)

It was the intent that `entries` would be a `List[CallCachingEntry]` and all the others would be `List[Seq[_]]` or `List[List[_]]` of parallel dependent entities.